src/file/file.rpc.ts

```diff
@@ -1,7 +1,14 @@
-import { FileService } from './file.service';
-import { IFile } from './file.interface';
+const apm = require('elastic-apm-node').start({
+  serviceName: 'file-service',
+  secretToken: '',
+  serverUrl: `${elasticURL}`,
+});
+
 import { ObjectID } from 'mongodb';
 import { GrpcHealthCheck, HealthCheckResponse, HealthService } from 'grpc-ts-health-check';
+import { FileService } from './file.service';
+import { IFile } from './file.interface';
+import { elasticURL } from '../config';
 
 const grpc = require('grpc');
```

Comment:
Use grpc types...
Here: import * as grpc from 'grpc';

Reply:
This includes using `grpc`'s types for params like `call`, `file_proto`, `server`, `rpcFunction`, `callback` and probably more.

Reply:
All I can do is give the functions type 'Function'

Reply:
@Shahar-Y Type `grpc.ServerUnaryCall` exists for `call`.
For `callback` there's type `grpc.requestCallback`, and there are others too...

Reply:
Done.

Reply:
@Shahar-Y `file_proto` is still `any`, so it ain't done.